lib/src/index.js

```diff
@@ -1,4 +1,12 @@
-const Navigation = require('./Navigation');
+import Navigation from './Navigation';
+import { FloatingActionButton } from './view/FloatingActionButton';
+import { CoordinatorLayout } from './view/CoordinatorLayout';
+
+export {
+  Navigation,
+  FloatingActionButton,
+  CoordinatorLayout
+};
+
+export default Navigation;
 
-module.exports = Navigation;
-module.exports.default = Navigation;
```

Comment:
undo this change. Navigation should be the only entry point. these 2 classes should be static properties of the internal Navigation class